src/Numerics/project.json

```diff
@@ -0,0 +1,97 @@
+﻿{
```

Comment:
The mere presence of a project.json file used to cause havoc on the other projects. I didn't run into any issues yet though, neither in the build nor in VS. Has this been fixed then somehow? That would be great, this was a major blocker up to now.